plugins/inputs/unbound/unbound_windows.go

```diff
@@ -0,0 +1,3 @@
+// +build windows
```

Comment:
Let's leave this file out, if somehow you have unbound-control on Windows then have at it.